slices/coreutils.yaml

```diff
@@ -422,6 +421,155 @@ slices:
     contents:
       /usr/bin/printf:
 
+  basename:
+    essential:
+      - libc6_libs
+    contents:
+      /usr/bin/basename:
+
+  chmod:
+    essential:
+      - libc6_libs
+    contents:
+      /usr/bin/chmod:
+
+  chown:
+    essential:
+      - libc6_libs
+    contents:
+      /usr/bin/chown:
+
+  cut:
+    essential:
+      - libc6_libs
+    contents:
+      /usr/bin/cut:
+
+  date:
+    essential:
+      - libc6_libs
+    contents:
+      /usr/bin/date:
+
+  dd-utility:
```

Comment:
I'm guessing from the pattern here that this is because 2-letter slice names aren't allowed?

Could we call each of these `{util}-bin` instead?